### PR TITLE
[ES] osu!wiki Contribution Guide

### PR DESCRIPTION
**Pull request by axuz**
Added Spanish translation to the osu!wiki Contribution Guide

### Status

- [ ] WIP
- [ ] Pending Review
- [x] Finished

### CODE REVIEW

File: wiki/osu!wiki_Contribution_Guide/es.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+11\. Ve el apartado [Revisiones](#reviews) y [Unión](#merging) abajo.
+
+### Revisiones
+
+Una vez que hayas creado tu Pull Request, otros editores de osu!wiki pueden revisar tus cambios para ayudarte en localizar algunos errores que hayas cometido. **Necesitarás mantenerte al día con las revisiones,** de otra manera, ¡tu Pull Request se cancelará! Si necesitas que alguien revise tu Pull Request, puedes preguntar a otros editores de la osu!wiki en el [chat de osu!dev en Discord](https://discord.gg/ppy) o en los comentarios de GitHub.
+
+### Unión 
+
+Para que tus cambios aparezcan en osu!wiki, tu Pull Request necesita ser "unido". Una vez que tu Pull Request haya sido revisado y aprovado, puedes usar la sección de comentarios en GitHub para preguntar a alguien que lo una, o haz una pregunta en el [chat de osu!dev en Discord](https://discord.gg/ppy). Una vez que haya sido unido, tus cambios tardarán hasta cinco horas en aparecer en osu!wiki.
```
Comment (KaiMinCL) [inline wiki/osu!wiki_Contribution_Guide/es.md]: `en osu!wiki` -> `en la osu!wiki`
`tu Pull Request necesita ser "unido".` -> `tu "pull request" necesita ser unida.`
`una vez que tu Pull Request haya sido revisado y aprovado.` -> `una vez que tu "pull request" haya sido revisada y aprovada.`
`alguien que lo una` -> `alguien que la una`
`Una vez que haya sido unido` -> `Una vez que haya sido unida`

File: wiki/osu!wiki_Contribution_Guide/es.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+10\. Cuando estés listo, haz clic en `Create pull request`.
+
+![](img/new-pull-request.jpg "Perfecto")
+
+11\. Ve el apartado [Revisiones](#reviews) y [Unión](#merging) abajo.
+
+### Revisiones
+
+Una vez que hayas creado tu Pull Request, otros editores de osu!wiki pueden revisar tus cambios para ayudarte en localizar algunos errores que hayas cometido. **Necesitarás mantenerte al día con las revisiones,** de otra manera, ¡tu Pull Request se cancelará! Si necesitas que alguien revise tu Pull Request, puedes preguntar a otros editores de la osu!wiki en el [chat de osu!dev en Discord](https://discord.gg/ppy) o en los comentarios de GitHub.
```
Comment (KaiMinCL) [inline wiki/osu!wiki_Contribution_Guide/es.md]: `Pull Request` -> `"pull request"`
`¡tu Pull Request se cancelará!` -> `!Tu pull pequest se cancelará!`

File: wiki/osu!wiki_Contribution_Guide/es.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+3\. Haz clic en `Fork` de nuevo para ir a tu clon.
+
+4\. Si ves algo similar a la imágen significa que hiciste un clon del repositorio de `ppy/osu-wiki` y estás en ese mismo clon.
+
+![](img/forked.jpg)
+
+5\. Continúa con la [Sincronización de tu clon](#syncing-your-fork).
+
+### Sincronización de tu clon
```
Comment (KaiMinCL) [inline wiki/osu!wiki_Contribution_Guide/es.md]: `Sincronización de tu clon` -> `Sincronizando tu clon` (Seria una traducción mas exacta)

**Comment by axuz** [comment]
He corregido todos los errores. Si alguien encuentra otro error por favor avisen!

**Comment by KaiMinCL** [comment]
Cambiar `Clon` (y sus derivados) por `bifurcación` (y sus derivados) Ya que la traduccion de Fork es Bifurcación o rama (https://es.wikipedia.org/wiki/Bifurcaci%C3%B3n_(desarrollo_de_software))

**Comment by axuz** [comment]
Gracias, agregué los cambios que dijiste. Si alguien encuentra más errores por favor avisen

**Comment by axuz** [comment]
Gracias de nuevo, ya los corregí. si alguien encuentra más errores por favor avisar

**Comment by axuz** [comment]
This is now fully translated. I rechecked it and nothing else is missing.

Everything is translated correctly.

File: wiki/osu!wiki_Contribution_Guide/es.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
+# Guía de Contribución de la osu!wiki
```
Comment (Joehuu) [inline wiki/osu!wiki_Contribution_Guide/es.md]: `osu!wiki` -> `osu! wiki`
Can you do that for every instance? Including the name of the folder to `osu!_wiki_Contribution_Guide`. A recent PR was pushed with these minor changes.

Comment (axuz) [reply wiki/osu!wiki_Contribution_Guide/es.md]: Fixed! Do I have to create a new PR to change the folder location?

Comment (Joehuu) [reply wiki/osu!wiki_Contribution_Guide/es.md]: No, you can do it on this PR. Hope you know what to do.

Comment (axuz) [reply wiki/osu!wiki_Contribution_Guide/es.md]: hmm... I'm a little bit stuck. I tried changing the file path but the exclamation mark in the folder's name (!) gets replaced by a dash (-), so it's not letting me change the directory. Is there another way to change it? I'm sorry, I'm still a newbie at this kind of things :P

Comment (Joehuu) [reply wiki/osu!wiki_Contribution_Guide/es.md]: It seems that you are using the web interface. The web is limited to many things like deleting/creating folders.

Doing it locally will work and is better if you plan on making more PRs. You have to download Github Desktop and an editor. It prefers you to download Atom, but you can use other editors. I use Atom btw.

Comment (axuz) [reply wiki/osu!wiki_Contribution_Guide/es.md]: thanks for the advice!

**Comment by axuz** [comment]
changed the file path to the new one and this seems to be ready
if anyone finds any more mistakes please comment :smile:

File: wiki/osu!wiki_Contribution_Guide/es.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
+# Guía de Contribución de la osu! wiki
```
Comment (Joehuu) [inline wiki/osu!wiki_Contribution_Guide/es.md]: Remove this duplicate file.

File: wiki/osu!_wiki_Contribution_Guide/es.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+### Abrir un "pull request"
+
+1\. Ve al [repositorio `ppy/osu-wiki`](https://github.com/ppy/osu-wiki).
+
+2\. Si fuiste lo suficientemente rápido, es posible que veas este cartel amarillo
+
+![](img/github-recent.jpg)
+
+3\. Si lo ves, haz clic en el botón `Compare & pull request` (saltar hasta el paso 8). Si no, haz clic en  el botón `New pull request`(continúa con el paso 4).
```
Comment (Joehuu) [inline wiki/osu!_wiki_Contribution_Guide/es.md]: Between en and el, there are two spaces. Only shows in code. Shows as one space in preview.

**Comment by axuz** [comment]
thanks.
if anybody else finds any other mistakes please comment :smiley:

**Pull request merged**